### DISCUSSION TITLE
Small performance mod missing from the list

### DISCUSSION THREAD

**Discussion by Serotonone** [TheUsefulLists/UsefulMods#184] [General]
found this gem while googling random stuff:

https://www.curseforge.com/minecraft/mc-mods/mcmt-multithreading

This is a mod, that multithreads minecraft's tick execution. Crazy, I know. It's brought to you by the dark forces of coremodding, and works only as well as I've tested it

This mod alters the serverside processing (be it on a dedicated server or the single player integrated one) and dispaches all worlds, entities, tile entites and environment ticks to a thread pool, allowing for the parallel execution of minecraft. Under testing this shows about a 20+% improvement in tick times Enabled vs Disabled in single player. Note that uninstalled vs disabled appears to be similar but will have different time charicteristics as this does muck around with lots of the game's execution. In single player I've seen this be closer to 50% on average, and in multiplayer it can possibly be even higher.

Theoretically if this is developed enough (i.e. to the point of stability), it should allow more players in a single instance of a server; or more chunks to be loaded and processed in a modded game.

Note: This mod only mulithreads tick execution; it will probably not help with framerate or anything like that.

**Comment by spotishii** [comment]
No, this is awesome! I keep freezing because of ticking entities so this is a must have mod for me to rule out the crashing.
